sabnzbd/notifier.py

```diff
@@ -265,6 +270,67 @@ def send_prowl(title, msg, notification_type, force=False, test=None):
     return ""
 
 
+def send_apprise(title, msg, notification_type, force=False, test=None):
+    """send apprise message"""
+    logging.debug("Sending Apprise notification")
+    if test:
+        urls = test.get("apprise_urls")
+    else:
+        urls = sabnzbd.cfg.apprise_urls()
+
+    # Notification mapper
+    n_map = {
+        # Startup/Shutdown
+        "startup": apprise.common.NotifyType.INFO,
+        # Pause/Resume
+        "pause_resume": apprise.common.NotifyType.INFO,
+        # Added NZB
+        "download": apprise.common.NotifyType.INFO,
+        # Post-processing started
+        "pp": apprise.common.NotifyType.INFO,
+        # Job finished
+        "complete": apprise.common.NotifyType.SUCCESS,
+        # Job failed
+        "failed": apprise.common.NotifyType.FAILURE,
+        # Warning
+        "warning": apprise.common.NotifyType.WARNING,
+        # Error
+        "error": apprise.common.NotifyType.FAILURE,
+        # Disk full
+        "disk_full": apprise.common.NotifyType.WARNING,
+        # Queue finished
+        "queue_done": apprise.common.NotifyType.INFO,
+        # User logged in
+        "new_login": apprise.common.NotifyType.INFO,
+        # Other Messages
+        "other": apprise.common.NotifyType.INFO,
+    }
+
+    # Prepare our Asset Object
+    asset = apprise.AppriseAsset(
+        app_id="SABnzbd",
+        app_desc="SABnzbd Notification",
+        app_url="http://sabnzbd.org/",
+        image_path_mask=os.path.join(os.path.dirname(__file__), "apprise", "apprise-{TYPE}.png"),
+        image_url_mask="https://raw.githubusercontent.com/sabnzbd/sabnzbd/develop/sabnzbd/apprise/apprise-{TYPE}.png",
```

Comment:
Should we maybe host these at sabnzbd.org?
https://github.com/sabnzbd/sabnzbd.github.io/tree/master/images/icons

Reply:
Sure we can, let me know the URL i should use.  Even using the one you just shared now:
- `https://github.com/sabnzbd/sabnzbd.github.io/tree/master/images/icons` becomes `https://raw.githubusercontent.com/sabnzbd/sabnzbd.github.io/master/images/icons/` for direct reference.

Perhaps you man the url:
- `https://github.com/sabnzbd/sabnzbd.github.io/tree/master/images/icons` (as it is now) should change to: `https://sabnzbd.org/images/icons/` ?

So ... ?
```python
    # Prepare our Asset Object
    asset = apprise.AppriseAsset(
        app_id="SABnzbd",
        app_desc="SABnzbd Notification",
        app_url="http://sabnzbd.org/",
        image_path_mask=os.path.join(os.path.dirname(__file__), "apprise", "apprise-{TYPE}.png"),
        image_url_mask="https://sabnzbd.org/images/icons/apprise/apprise-{TYPE}.png",
        image_url_logo="https://sabnzbd.org/images/icons/"
        "apple-touch-icon-120x120-precomposed.png/apple-touch-icon-180x180-precomposed.png",
    )
```

Reply:
Yes that last one is what I mean. If you open a PR to add them to https://github.com/sabnzbd/sabnzbd.github.io/tree/master/images/icons then it will automatically be available at sabnzbd.org.

Reply:
So just to follow up with this, we still need the images in this repository (somewhere), because notifications differ from where they get their data from.  For Discord, Slack, etc, no problem,; they'll get their image from the web.  But other services actually need their image uploaded as part of the payload for it's assembly.  So the Apprise plugins are smart enough to determine if they need to go to the cloud for the `error`/`success`/`info`/`warning` message or to look on the local disk.  Otherwise, it looks like we're good to go I see you merged [abnzbd.github.io/309](https://github.com/sabnzbd/sabnzbd.github.io/pull/309)